src/calculateRentalCost.js

```diff
@@ -28,7 +28,20 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dailyCost = 40;
+  const discountThreeDays = 20;
+  const discountSevenDays = 50;
+  const lessThanThreeDaysCost = dailyCost * days;
+  const lessThanSevenDayCost = lessThanThreeDaysCost - discountThreeDays;
+  const moreThanSevenDayCost = lessThanThreeDaysCost - discountSevenDays;
+
+  if (days < 3) {
+    return lessThanThreeDaysCost;
+  } else if (days > 2 && days < 7) {
```

Comment:
Pay attention to all comments

Reply:
Also, you don't need to check if there is less than 2 days, cause you check it in previous 'if' clause
```suggestion
  } else if ( days < 7) {
```